### ISSUE TITLE
Getting errors when trying to run from Windows binaries

### ISSUE THREAD

**Issue by madshall** [bosun-monitor/bosun#1339]
I'm trying to make Bosun working with our InfluxDB instance that receives data from Scala project via HTTP. As I assume I don't need scollector to be installed as we already collect all the data we need. However, we need an easily configurable alerting system to be included. I installed Bosun, I have my InfluxDB service running and I also have Grafana being running as well. Is it even possible to make all of those work together? Prior to any questions regarding this integration I tried at least to build and run Bosun to see what it is and here's what I see when running w.sh on my Windows 7 machine:

```
2015/09/22 10:46:02 info: bolt.go:123: RestoreState
2015/09/22 10:46:02 error: bolt.go:151: metadata unknown bucket: bindata
2015/09/22 10:46:02 error: bolt.go:154: metadata-metric unknown bucket: bindata

2015/09/22 10:46:02 error: bolt.go:163: metric unknown bucket: bindata
2015/09/22 10:46:02 error: bolt.go:166: tagk unknown bucket: bindata
2015/09/22 10:46:02 error: bolt.go:169: tagv unknown bucket: bindata
2015/09/22 10:46:02 error: bolt.go:173: notifications unknown bucket: bindata
2015/09/22 10:46:02 error: bolt.go:176: silence unknown bucket: bindata
2015/09/22 10:46:02 error: bolt.go:179: incidents unknown bucket: bindata
2015/09/22 10:46:02 error: bolt.go:182: errors unknown bucket: bindata
2015/09/22 10:46:02 error: bolt.go:193: status unknown bucket: bindata
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x8 pc=0x7721a0]

goroutine 1 [running]:
bosun.org/_third_party/github.com/boltdb/bolt.(*Bucket).Delete(0x0, 0xc082065198, 0xa, 0x20, 0x0, 0x0)
        C:/go_projects/src/bosun.org/_third_party/github.com/boltdb/bolt/bucket.go:307 +0x40
bosun.org/cmd/bosun/sched.deleteKey.func1(0xc082255ba0, 0x0, 0x0)
        C:/go_projects/src/bosun.org/cmd/bosun/sched/bolt.go:318 +0x101
bosun.org/_third_party/github.com/boltdb/bolt.(*DB).Update(0xc0821cad00, 0xc082065270, 0x0, 0x0)
        C:/go_projects/src/bosun.org/_third_party/github.com/boltdb/bolt/db.go:547 +0xba
bosun.org/cmd/bosun/sched.deleteKey(0xc0821cad00, 0xe1a7d0, 0xa, 0x0, 0x0)
        C:/go_projects/src/bosun.org/cmd/bosun/sched/bolt.go:319 +0x5a
bosun.org/cmd/bosun/sched.(*Schedule).RestoreState(0x13efd80, 0x0, 0x0)
        C:/go_projects/src/bosun.org/cmd/bosun/sched/bolt.go:251 +0x1af4
bosun.org/cmd/bosun/sched.(*Schedule).Load(0x13efd80, 0xc0820fb680, 0x0, 0x0)
        C:/go_projects/src/bosun.org/cmd/bosun/sched/sched.go:527 +0x98
bosun.org/cmd/bosun/sched.Load(0xc0820fb680, 0x0, 0x0)
        C:/go_projects/src/bosun.org/cmd/bosun/sched/sched.go:512 +0x40
main.main()
        c:/go_projects/src/bosun.org/cmd/bosun/main.go:102 +0x590

goroutine 6 [syscall]:
os/signal.loop()
        c:/go/src/os/signal/signal_unix.go:22 +0x1f
created by os/signal.init.1
        c:/go/src/os/signal/signal_unix.go:28 +0x3e

goroutine 19 [sleep]:
time.Sleep(0x12a05f200)
        c:/go/src/runtime/time.go:59 +0x107
bosun.org/metadata.collectMetadata()
        C:/go_projects/src/bosun.org/metadata/metadata.go:193 +0x3c
created by bosun.org/metadata.Init
        C:/go_projects/src/bosun.org/metadata/metadata.go:186 +0xc8
exit status 2
```

**Comment by nathanielc** [comment]
I just got the same panic on Linux using master. Here is my config that reproduces the error:

```
httpListen = :8070
timeAndDate = 202,75,179,136
checkFrequency = 5m
stateFile = ./bosun.state

notification default {
  print = true
}

```

The file `./bosun.state` does not exist as this is the first time I am starting bosun.

Ran `./w.sh` on a clean pull from master.

Looks like the `RestoreState` method on the `Schedule` doesn't know how to gracefully fail if there isn't any state to restore. https://github.com/bosun-monitor/bosun/blob/master/cmd/bosun/sched/bolt.go#L119
